pkg/deploy/deploy.go

```diff
@@ -281,12 +281,6 @@ func manageResource(ctx context.Context, cli client.Client, res *resource.Resour
 	if err == nil {
 		// when resource is found
 		if enabled {
-			// Exception to not update kserve with managed annotation
-			// do not reconcile kserve resource with annotation "opendatahub.io/managed: false"
-			// TODO: remove this exception when we define managed annotation across odh
-			if found.GetAnnotations()[annotations.ManagedByODHOperator] == "false" && componentName == "kserve" {
-				return nil
-			}
```

Comment:
looks like the changes in this file do not belong to this PR ?

Reply:
this is wrong, mixed with another PR, i will revert it